source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -372,8 +374,8 @@ information:
 - The BSON Type of each field (only required for deterministically
   encrypted fields)
 
-.. admonition::Automatic Client-Side Field Level Encryption JSON Schema Does Not Support Document Validation
-   :important:
+.. admonition:: Automatic Client-Side Field Level Encryption JSON Schema Does Not Support Document Validation
```

Comment:
Is it possible to include a link here to the manual page on automatic CSFLE? I realize you were just fixing whitespace here, but I personally had to google automatic vs. normal CSFLE when I read this.

Reply:
It would also help this feel slightly less like a Demented randOm Capitalization statement

Reply:
Great feedback. I will omit "Automatic" as it does not belong. When writing this, I was probably in the frame of mind of emphasizing that CSFLE is automatic. While it could always be manually implemented, this feature is our "official" implementation.
From: https://docs.mongodb.com/manual/core/security-encryption-at-rest/#application-level-encryption

> New in version 4.2: MongoDB 4.2-series drivers provides a client-side field level encryption framework. For more information, see Client-Side Field Level Encryption.
> To encrypt full documents, write custom encryption and decryption routines or use a commercial solution.

Reply:
(I wish the server doc linked in this admonition did not contain automatic in the title)

Reply:
Sounds good, slightly more sane now :)